src/Agent.Worker/Handlers/Handler.cs

```diff
@@ -295,5 +295,14 @@ protected void AddPrependPathToEnvironment()
                 AddEnvironmentVariable(Constants.PathVariable, newPath);
             }
         }
+
+        protected void RemovePSModulePathFromEnvironment()
```

Comment:
Should it be inside the `PowerShell3Handler`? Or, why we are not use it in the `PowerShellExeHandler`?

Reply:
Added it to the `PowerShellExeHandler` as well, thanks.